src/firefly/js/ui/dynamic/EmbeddedPositionSearchPanel.jsx

```diff
@@ -48,7 +93,12 @@ export function EmbeddedPositionSearchPanel({
                                             } ) {
 
     const {groupKey}= useContext(FieldGroupCtx);
-    const [getConeAreaOp, setConeAreaOp] = useFieldGroupValue(CONE_AREA_KEY);
+    const {searchTypeKey=CONE_AREA_KEY}= slotProps.spatialSearch ?? {};
+    const {targetKey=DEF_TARGET_PANEL_KEY}= slotProps.targetPanel ?? {};
+    const {polygonKey=DEFAULT_POLYGON_KEY, }= slotProps.polygonField ?? {};
+    const {sizeKey= DEFAULT_SIZE_KEY, min= 1 / 3600, max= 1}= slotProps.sizeInput ?? {};
```

Comment:
I did that because I added `searchTypeKey` that needs to be destructed early on for use in `useFieldGroupValue`. And then rest 3 are also most important keys in this component. But I don't feel strongly about it, I've moved them back to their original position.